Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -2369,6 +2366,79 @@ void MainWindow::propagate_action()
     }
   }
 }
+
+void MainWindow::on_actionSa_ve_Scene_as_Script_triggered()
+{
+  QString filename =
+      QFileDialog::getSaveFileName(this,
+                                   "Save the Scene as a Script File",
+                                   last_saved_dir,
+                                   "*.js");
```

Comment:
The filter is not right, and not not work as expected. It should be `Qt Script files (*.js)`.